toolkit/types/cpe/unbind.go

```diff
@@ -64,16 +64,24 @@ func UnbindURI(s string) (WFN, error) {
 		if i == 5 && strings.HasPrefix(c, "~") {
 			attrs := [...]Attribute{Edition, SwEdition, TargetSW, TargetHW, Other}
 			for i, c := range strings.SplitN(c, `~`, 6)[1:] {
-				r.Attr[attrs[i]].unbindURI(&b, c)
+				if err := r.Attr[attrs[i]].unbindURI(&b, c); err != nil {
+					return WFN{}, fmt.Errorf("cpe: %v: %w", attrs[i], err)
+				}
 			}
 			continue
 		}
-		r.Attr[attrs[i]].unbindURI(&b, c)
+		if err := r.Attr[attrs[i]].unbindURI(&b, c); err != nil {
+			return WFN{}, fmt.Errorf("cpe: %v: %w", attrs[i], err)
+		}
 	}
 	return r, r.Valid()
 }
 
-func (v *Value) unbindURI(b *strings.Builder, s string) {
+func (v *Value) unbindURI(b *strings.Builder, s string) error {
+	// From the characters that should be escaped, Table 6-1.
```

Comment:
I can't find the Table 6-1 (or [here](https://nvlpubs.nist.gov/nistpubs/Legacy/IR/nistir7696.pdf) it seems to show something different), would it be possible to add the link as well?

Reply:
That's the matching specification, not the naming spec. I'll add a link.